scripts/index.js

```diff
@@ -1 +1,28 @@
-// add your solutions here
\ No newline at end of file
+// add your solutions here
+arrayOne = [1,2,3,4,5,6,7,8,9,10,-1,-2,-3,20,99,97,200, 220, 12];
```

Comment:
Make sure to use `const` or `let` when you assign to a variable for the first time. Check out http://tobyho.com/2011/10/25/js-accidental-global-variables/ for more info.